lib/oli_web/router.ex

```diff
@@ -742,40 +742,56 @@ defmodule OliWeb.Router do
   end
 
   ### Sections - Student Course Delivery
-  scope "/sections", OliWeb do
+  scope "/sections/:section_slug", OliWeb do
     pipe_through([
       :browser,
       :require_section,
+      :delivery,
       :delivery_protected,
       :maybe_gated_resource,
       :enforce_enroll_and_paywall,
       :pow_email_layout
     ])
 
-    get("/:section_slug/overview", PageDeliveryController, :index)
-    get("/:section_slug/container/:revision_slug", PageDeliveryController, :container)
-    get("/:section_slug/page/:revision_slug", PageDeliveryController, :page)
-    get("/:section_slug/page/:revision_slug/page/:page", PageDeliveryController, :page)
-    get("/:section_slug/page/:revision_slug/attempt", PageDeliveryController, :start_attempt)
+    get("/overview", PageDeliveryController, :index)
+
+    live("/learning_objectives", Delivery.InstructorDashboard.LearningObjectivesLive)
+    live("/students", Delivery.InstructorDashboard.StudentsLive)
+    live("/content", Delivery.InstructorDashboard.ContentLive)
+    live("/discussions", Delivery.InstructorDashboard.DiscussionLive)
+
+    get("/container/:revision_slug", PageDeliveryController, :container)
+    get("/page/:revision_slug", PageDeliveryController, :page)
+    get("/page/:revision_slug/page/:page", PageDeliveryController, :page)
+    get("/page/:revision_slug/attempt", PageDeliveryController, :start_attempt)
 
     get(
-      "/:section_slug/page/:revision_slug/attempt/:attempt_guid/review",
+      "/page/:revision_slug/attempt/:attempt_guid/review",
       PageDeliveryController,
       :review_attempt
     )
   end
 
   ### Sections - Preview
-  scope "/sections/:section_slug/preview/", OliWeb do
+  scope "/sections/:section_slug/preview", OliWeb do
     pipe_through([
       :browser,
       :require_section,
       :authorize_section_preview,
-      :delivery_and_admin,
```

Comment:
Dropping `:delivery_and_admin` here seems to have the effect of requiring me, logged in as an admin, to enroll as a student to be able to access these instructor specific routes.

Reply:
restored delivery_and_admin to pipeline